.checkstyle/checkstyle.xml

```diff
@@ -87,21 +87,23 @@
         </module>
 
         <!-- code quality -->
-        <module name="MethodLength"/>
+        <module name="MethodLength">
```

Comment:
Can't we just suppress these with annotations directly in the code? That is much better then rewriting the rules for all methods.

Reply:
Make sense, let me change it